src/test/java/frc/robot/util/ControllerHelpersTest.java

```diff
@@ -8,8 +8,8 @@
 public class ControllerHelpersTest {
   @Test
   void circularDiscCoordinateTest() {
-    var result = ControllerHelpers.fromCircularDiscCoordinates(-0.68, -0.58);
-    var expected = new Translation2d(-0.78, -0.7);
+    var result = ControllerHelpers.fromCircularDiscCoordinates(-0.9, -0.9);
```

Comment:
can we add one or two other tests that check the deadbanding behavior when close to 0 and a regular input (not 0.9)